packages/angular-output-target/src/generate-angular-component.ts

```diff
@@ -1,130 +1,128 @@
 import { dashToPascalCase, normalizePath } from './utils';
 import type { ComponentCompilerMeta } from '@stencil/core/internal';
 
-export const createComponentDefinition = (
-  componentCorePackage: string,
-  distTypesDir: string,
-  rootDir: string,
-  includeImportCustomElements: boolean = false,
-  customElementsDir: string = 'components'
-) => (cmpMeta: ComponentCompilerMeta) => {
-  // Collect component meta
-  const inputs = [
-    ...cmpMeta.properties.filter((prop) => !prop.internal).map((prop) => prop.name),
-    ...cmpMeta.virtualProperties.map((prop) => prop.name),
-  ].sort();
-  const outputs = cmpMeta.events.filter((ev) => !ev.internal).map((prop) => prop);
-  const methods = cmpMeta.methods.filter((method) => !method.internal).map((prop) => prop.name);
-
-  // Process meta
-  const hasOutputs = outputs.length > 0;
-
-  // Generate Angular @Directive
-  const directiveOpts = [
-    `selector: \'${cmpMeta.tagName}\'`,
-    `changeDetection: ChangeDetectionStrategy.OnPush`,
-    `template: '<ng-content></ng-content>'`,
-  ];
-  if (inputs.length > 0) {
-    directiveOpts.push(`inputs: ['${inputs.join(`', '`)}']`);
-  }
-
-  const tagNameAsPascal = dashToPascalCase(cmpMeta.tagName);
-
-  const outputsInterface: Set<string> = new Set();
-  const outputReferenceRemap: { [p: string]: string } = {};
-
-  outputs.forEach((output) => {
-    Object.entries(output.complexType.references).forEach(([reference, refObject]) => {
-      // Add import line for each local/import reference, and add new mapping name.
-      // `outputReferenceRemap` should be updated only if the import interface is set in outputsInterface,
-      // this will prevent global types to be remapped.
-      const remappedReference = `I${cmpMeta.componentClassName}${reference}`;
-      if (refObject.location === 'local' || refObject.location === 'import') {
-        outputReferenceRemap[reference] = remappedReference;
-        let importLocation: string = componentCorePackage;
-        if (componentCorePackage !== undefined) {
-          const dirPath = includeImportCustomElements ? `/${customElementsDir || 'components'}` : '';
-          importLocation = `${normalizePath(componentCorePackage)}${dirPath}`;
+export const createComponentDefinition =
```

Comment:
These changes are a little more palatable if you "ignore whitespace" in GH's diff view